src/Avalonia.Themes.Simple/Controls/ScrollViewer.xaml

```diff
@@ -10,48 +10,23 @@
               RowDefinitions="*,Auto">
           <ScrollContentPresenter Name="PART_ContentPresenter"
                                   Padding="{TemplateBinding Padding}"
-                                  Background="{TemplateBinding Background}"
-                                  CanHorizontallyScroll="{TemplateBinding CanHorizontallyScroll}"
-                                  CanVerticallyScroll="{TemplateBinding CanVerticallyScroll}"
-                                  Content="{TemplateBinding Content}"                                  
                                   HorizontalSnapPointsType="{TemplateBinding HorizontalSnapPointsType}"
                                   VerticalSnapPointsType="{TemplateBinding VerticalSnapPointsType}"
                                   HorizontalSnapPointsAlignment="{TemplateBinding HorizontalSnapPointsAlignment}"
                                   VerticalSnapPointsAlignment="{TemplateBinding VerticalSnapPointsAlignment}"
-                                  Extent="{TemplateBinding Extent,
-                                                           Mode=TwoWay}"
-                                  IsScrollChainingEnabled="{TemplateBinding IsScrollChainingEnabled}"
-                                  Viewport="{TemplateBinding Viewport,
-                                                             Mode=TwoWay}"
-                                  Offset="{TemplateBinding Offset,
-                                                           Mode=TwoWay}">
+                                  Background="{TemplateBinding Background}">
             <ScrollContentPresenter.GestureRecognizers>
-              <ScrollGestureRecognizer CanHorizontallyScroll="{TemplateBinding CanHorizontallyScroll}"
-                                       CanVerticallyScroll="{TemplateBinding CanVerticallyScroll}" />
+              <ScrollGestureRecognizer CanHorizontallyScroll="{Binding CanHorizontallyScroll, ElementName=PART_ContentPresenter}"
+                                       CanVerticallyScroll="{Binding CanVerticallyScroll, ElementName=PART_ContentPresenter}"
+                                       IsScrollInertiaEnabled="{Binding IsScrollInertiaEnabled, RelativeSource={RelativeSource TemplatedParent}}" />
```

Comment:
This property was not being set. I assumed that this was a mistake and added a binding.